biscuit-auth/src/datalog/mod.rs

```diff
@@ -743,12 +743,17 @@ pub struct RunLimits {
     pub max_time: Duration,
 }
 
+#[cfg(test)]
+const DEFAULT_DURATION_MILLIS: u64 = 1;
+#[cfg(not(test))]
+const DEFAULT_DURATION_MILLIS: u64 = 10_000;
```

Comment:
shouldn't they be reversed? The longer one in tests?

Reply:
🤦

Reply:
(i had removed the commit anyway but i’ll give another try)